packages/react-sdk/src/components/index.ts

```diff
@@ -157,7 +169,11 @@ export const defaultPropsMetas: Record<string, WsComponentPropsMeta> = {
   RadioButton: RadioButtonPropsMeta,
   CheckboxField: CheckboxFieldPropsMeta,
   Checkbox: CheckboxPropsMeta,
-};
+  Tabs: TabsRootPropsMeta,
```

Comment:
Prefix with radix? Like RadixTabs

Reply:
Was thinking about it as well, especially if we go with "Radix UI" as a category, need to discuss this with Taylor. Ideally this would be part of a UI that lets user install those components but we in the end still need to decide how we represent the section of installed components or even packages. So the closest thing I can think of right now is to just put all Radix UI components into a section/category

Reply:
Label can still be without prefix. I mean would be good to prefix the identifier so we could avoid fatal conflicts if someone adds another library with same component name.